assets/src/edit-story/components/colorPicker/patternTypePicker.js

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { __ } from '@web-stories-wp/i18n';
+import {
+  ToggleButton,
+  Icons,
+  BUTTON_SIZES,
+  BUTTON_VARIANTS,
+} from '@web-stories-wp/design-system';
+
+/**
+ * Internal dependencies
+ */
+
+const HEADER_FOOTER_HEIGHT = 52;
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  height: ${HEADER_FOOTER_HEIGHT}px;
+  padding: 8px 16px 16px;
+  position: relative;
+  gap: 8px;
+`;
+
+function PatternTypePicker({ type, setToGradient, setToSolid }) {
+  const setToLinear = useCallback(
+    () => setToGradient('linear'),
+    [setToGradient]
+  );
+  const setToRadial = useCallback(
+    () => setToGradient('radial'),
+    [setToGradient]
+  );
+
+  return (
+    <Wrapper>
+      <ToggleButton
+        aria-label={__('Solid pattern type', 'web-stories')}
+        onClick={setToSolid}
+        size={BUTTON_SIZES.SMALL}
+        variant={BUTTON_VARIANTS.SQUARE}
```

Comment:
Hm, these are using some other style than usual per Figma:
![Screenshot 2021-08-03 at 16 47 59](https://user-images.githubusercontent.com/3294597/128027910-b744dc29-4b2b-4a16-b751-d4e253fb5486.png)

They have just a border around instead of background as usually for buttons.

Reply:
I went with the design system buttons, as I've previously been told, that the design system trumps any individual designs. If a button/component/icon/color doesn't exist in the base system, it shouldn't be used (unless told otherwise).

Yes, I agree that it looks different, but it's the only button component we have.

Reply:
if we need to add a button variant I think we should. I've had this happen a few times now where the designs for the ds were just incomplete, which is fine, that should be a living system.

Reply:
This looks the most similar to what's in the design:
![Screenshot 2021-08-04 at 09 36 45](https://user-images.githubusercontent.com/3294597/128133587-9dae4950-b1f1-4fdd-880f-5d97992c7eba.png)

It might need just making sure that the active button has a lighter color.